.github/workflows/ci-markdown.yml

```diff
@@ -0,0 +1,34 @@
+# Much simpler CI for changes only to markdown documentation
+name: CI-markdown
+
+env:
+  SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
+
+# Trigger the workflow on the pushes that bors cares about
+on:
+  push:
+    branches:
+      - staging
+      - trying
+    # Only trigger if updating only docs
+    paths:
+      - '**.md'
+      - '!**.md'
```

Comment:
This seems wrong... Based on [this](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#onpushpull_requestpaths), this workflow will only trigger when any file with extension `md` is included, excluding any file with extension `md`, in other words, it will never run.

I couldn't find a way to achieve what we need using GHA filters. You could create a bash command that achieves it if no better solution comes to mind.

Reply:
You're right. I'll figure this out.